### ISSUE TITLE
Support for VPCs with multiple CIDR (Field cannot be changed: CIDR)

### ISSUE THREAD

**Issue by rdrgmnzs** [kubernetes/kops#3536]
AWS added support to add a secondary CIDR address to a VPC, however when trying to specify the secondary CIDR to be used by kops you get the following error: "Field cannot be changed: CIDR"

KOPS - Version 1.7.0 (git-e04c29d)
Also saw the same results when using the latest 1.8.0 beta.

Support for multiple CIDR was added to aws-sdk-go v1.10.34, where kops is currently using v1.6.10

[logoutput.txt](https://github.com/kubernetes/kops/files/1357063/logoutput.txt)

**Comment by mmacfadden** [comment]
I just ran into this same issue using 1.8,0. For reference here is the announcement from AWS:

https://aws.amazon.com/about-aws/whats-new/2017/08/amazon-virtual-private-cloud-vpc-now-allows-customers-to-expand-their-existing-vpcs/

It looks like the team is adding some improved functionality in KOPS 1.9.0 to use existing VPCs and Subnets. It would seem like this would be a good feature to have to round out that functionality. The subnets I created were in the "additional" CIDR associated with the VPC and I can't for the life of me get it to work.
